tests/FSharp.Data.Tests/CsvProvider.fs

```diff
@@ -563,3 +563,15 @@ let ``Having null in a cell should not fail saving to string (issue#978)`` () =
     data
     |> Stringify 
     |> ignore
+    
+[<Test>]
+let ``CsvFile.TryFindColumn returns only the first matching column``() =
```

Comment:
Don't forget to update the test name

Reply:
Yep. serves me right for rushing this between things today.